historical_surveillance/models.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+
+GRADE_CHOICES = [('grade k', 'Grade k'), ('grade 1', 'Grade 1'), ('grade 2', 'Grade 2'), ('grade 3', 'Grade 3'),
+                 ('grade 4', 'Grade 4'), ('grade 5', 'Grade 5'), ('grade 6', 'Grade 6'),
+                 ('form 1', 'Form 1'), ('form 2', 'Form 2'), ('form 3', 'Form 3'), ('form 4', 'Form 4'),
+                 ('form 5', 'Form 5'), ('form 6 L1', 'Form 6 L1'), ('form 6 L2', 'Form 6 L2')]
+
+CATEGORY_CHOICES = [('public primary', 'Public Primary'),
+                    ('public secondary', 'Public Secondary'),
+                    ('private primary', 'Private Primary'),
+                    ('private secondary', 'Private Secondary'),
+                    ('other', 'Other')]
+
+SEX_CHOICES = [('male', 'Male'),
+               ('female', 'Female')]
+
+
+# TO-DO
+# Enforce Foreign Key constraint on the schools and districts
+
+class District(models.Model):
+    objects = None
+    created_at = models.DateField(auto_now_add=True)
+    created_by = models.CharField(max_length=255)
+    district_code = models.CharField(max_length=50, blank=True)
+    district_name = models.CharField(max_length=50)
+    updated_at = models.DateField(auto_now_add=True)
+    updated_by = models.CharField(max_length=255, blank=True)
+
+    def __str__(self):
+        return str(self.district_name)
+
+
+class School(models.Model):
+    objects = None
+    created_at = models.DateField(auto_now_add=True)
+    created_by = models.CharField(max_length=255)
+    school_code = models.CharField(max_length=50, blank=True)
+    school_name = models.CharField(max_length=50)
+    district_name = models.ForeignKey(District, on_delete=models.CASCADE)
```

Comment:
This should be district instead of district name since its a fk to the district not the districts name